### PR TITLE
Change mta schema url to a valid one

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: Documentation/spec/docfx_document_schema.md [hunk 1]
```diff
@@ -186,7 +186,7 @@ Here's the schema to describe these operations:
 
 ```json
 {
-    "$schema": "https://github.com/dotnet/docfx/schemas/v1.0/schema.json#",
+    "$schema": "http://dotnet.github.io/docfx/schemas/v1.0/schema.json#",
```
Comment (vwxyzh) [inline Documentation/spec/docfx_document_schema.md]: `https`?

File: src/Microsoft.DocAsCode.Build.SchemaDriven/Validators/SchemaValidator.cs [hunk 2]
```diff
@@ -30,9 +30,23 @@ public static void Validate(DocumentSchema schema)
...
             }
         }
+
+        public static bool ValidateSchemaUrl(string url)
+        {
+            if (url == null || !Uri.TryCreate(url, UriKind.Absolute, out Uri uri))
+            {
+                return false;
+            }
+            if (uri.Host == SupportedMetaSchemaUri.Host && uri.LocalPath == SupportedMetaSchemaUri.LocalPath && (string.IsNullOrEmpty(uri.Fragment) || uri.Fragment == "#"))
```
Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.SchemaDriven/Validators/SchemaValidator.cs]: >uri.Host == SupportedMetaSchemaUri.Host [](start = 16, length = 39)

ignore case?

Comment (vicancy) [inline src/Microsoft.DocAsCode.Build.SchemaDriven/Validators/SchemaValidator.cs]: github does not ignore case

Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.SchemaDriven/Validators/SchemaValidator.cs]: >if (uri.Hos [](start = 12, length = 11)

can return the judgement directly?

Comment (superyyrrzz) [inline src/Microsoft.DocAsCode.Build.SchemaDriven/Validators/SchemaValidator.cs]: seems host name ignore case

---
In reply to: [133377688](https://github.com/dotnet/docfx/pull/1962#discussion_r133377688) [](ancestors = 133377688)

**Pull request merged**